src/inmanta/execute/runtime.py

```diff
@@ -710,6 +710,7 @@ def execute(self) -> None:
             self.resumer.resume(self.requires, self.resolver, self.queue)
         except RuntimeException as e:
             e.set_statement(self.resumer)
+            e.location = self.resumer.location
```

Comment:
shouldn't set_statement set the location?

Reply:
I'm not sure. It doesn't when it's already been set at least. Is this a bug?

Reply:
no, this is correct. So this is a hard override.